src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+            action=" " 
+            class="search-form firstSearchBar"
+            method="get" 
+            data-qa="big">
```

Comment:
fix indentation (2 tabs), also one class search-form is enough for the first form (the main one), but the second form will have second class (small, for example), so you will rewrite some styles related to sizes in this class
```suggestion
    <form
      action=" "
      class="search-form"
      method="get"
      data-qa="big"
    >
```